Description: Added AppStream metainfo XML with hardware provide info.
 This allow isenkram to propose this package when the USB dongle is
 present.
Author: Petter Reinholdtsen
Forwarded: no
Last-Update: 2024-07-17
---
Index: bluez-firmware-salsa/Makefile.am
===================================================================
--- bluez-firmware-salsa.orig/Makefile.am      2024-07-17 19:21:16.148904779 +0200
+++ bluez-firmware-salsa/Makefile.am   2024-07-17 19:21:16.144904740 +0200
@@ -11,6 +11,7 @@ firmware_DATA = \
 	st/STLC2500_R4_02_04.ptc
 
 EXTRA_DIST = $(firmware_DATA) \
+	org.bluez.firmware.metainfo.xml \
 	broadcom/BCM2033-2.13-FW.bin \
 	broadcom/BCM2033-2.14-FW.bin \
 	broadcom/BCM2033-2.15-FW.bin \
@@ -22,3 +23,6 @@ EXTRA_DIST = $(firmware_DATA) \
 MAINTAINERCLEANFILES = Makefile.in \
 	aclocal.m4 configure config.h.in \
 	depcomp missing install-sh mkinstalldirs
+
+dist_metainfo_DATA = org.bluez.firmware.metainfo.xml
+metainfodir = $(datarootdir)/metainfo
Index: bluez-firmware-salsa/org.bluez.firmware.metainfo.xml
===================================================================
--- /dev/null  1970-01-01 00:00:00.000000000 +0000
+++ bluez-firmware-salsa/org.bluez.firmware.metainfo.xml       2024-07-17 19:21:54.913286397 +0200
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.bluez.firmware</id>
+  <metadata_license>MIT</metadata_license>
+  <name>bluez-firmware</name>
+  <summary>Firmware for Bluetooth devices</summary>
+  <description>
+    <p>This firmware is required for operation of Bluetooth dongles
+    based on the Broadcom BCM203x and Raspberry Pi chipset.</p>
+  </description>
+  <provides>
+    <modalias>usb:v0C10p0000d**</modalias>
+  </provides>
+</component>
